assets/scss/components/_signup.scss

```diff
@@ -0,0 +1,14 @@
+.meetup-signUpModal {
+	.meetup-signUpModal-wrapper {
+		margin: $space $space-2;
+	}
+	.meetup-signUpModal-facebook {
+		background-color: $C_facebook;
+		&:hover {
+			background-color: $C_facebook;
+		}
+	}
+	.meetup-signUpModal-or {
+		text-transform: uppercase;
```

Comment:
can you use the class `.text--uppercase` instead?

Reply:
For some reason the "text--uppercase" class isn't working...